internal/service/kubernetes/kubernetes.go

```diff
@@ -71,6 +71,13 @@ func ResourceCluster() *schema.Resource {
 				Required:    true,
 				ForceNew:    true,
 			},
+			"private_node_groups": {
+				Description: "Enable private node groups. Private node groups requires network that is routed through NAT gateway.",
```

Comment:
```suggestion
				Description: "Enable private node groups. Private node groups require a network that is routed through NAT gateway.",
```

Reply:
Good catch :+1:  Thanks @villevsv-upcloud for the review.